taxi/views.py

```diff
@@ -1,3 +1,17 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
+from .models import Driver, Manufacturer, Car
 
-# Create your views here.
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_cars = Car.objects.count()
+    num_Manufacturers = Manufacturer.objects.count()
+
+    context = {
+    "num_drivers": num_drivers,
+    "num_cars": num_cars,
+    "num_Manufacturers": num_Manufacturers,
+    }
+
+    return render (request, "taxi/index.html", context=context)
```

Comment:
There is an unnecessary space between 'render' and '('. It's a minor issue, but it's better to keep the code style consistent. Consider removing the space.